package.json

```diff
@@ -29,7 +29,6 @@
         "prepare": "husky",
         "version": "npm version -m \"chore: release %s\"",
         "schemagen": "bash scripts/generate-json-schema.bash",
-        "schemacheck": "bash scripts/generate-json-schema.bash --check",
```

Comment:
Ah just noticed i did this in this PR. I noticed you can just do npm run schemagen -- --check instead.